### ISSUE TITLE
failed to write block to ancient store: append operation is out of order

### ISSUE THREAD

**Issue by meowsbits** [etclabscore/core-geth#184]
I'm not completely sure that this is reproducible since I saw it on a development branch, but putting here in case it pops up again.

```
WARN [09-15|11:38:11.735] Dropping unsynced node during fast sync  id=3cf8fa001b5e55ba conn=dyndial addr=94.130.68.239:30303   type=Geth/v1.9.3-stable-cfbb969d/linux-amd64/go1.11.5
WARN [09-15|11:38:25.539] Dropping unsynced node during fast sync  id=305492fd7c374a00 conn=dyndial addr=139.180.175.216:30303 type=Geth/v1.9.13-stable-cbc4ac26/linux-amd64/go1.13
WARN [09-15|11:38:30.605] Synchronisation failed, dropping peer    peer=1ee656d50bb79ad4 err=timeout
WARN [09-15|11:38:35.169] Rewinding blockchain                     target=1
INFO [09-15|11:38:35.383] Loaded most recent local header          number=1        hash="88e96d…406cb6" td=34351349760    age=5y2mo2w
INFO [09-15|11:38:35.383] Loaded most recent local full block      number=0        hash="d4e567…cb8fa3" td=17179869184    age=51y5mo1w
INFO [09-15|11:38:35.383] Loaded most recent local fast block      number=0        hash="d4e567…cb8fa3" td=17179869184    age=51y5mo1w
INFO [09-15|11:38:35.383] Loaded last fast-sync pivot marker       number=10867718
INFO [09-15|11:39:19.609] Imported new block headers               count=2047 elapsed=113.546ms number=2048     hash="7a6284…f9dc09" age=5y2mo1w  ignored=1
INFO [09-15|11:39:19.683] Downloader queue stats                   receiptTasks=0 blockTasks=1985 itemSize=109.57B throttle=8192
CRIT [09-15|11:39:19.683] Failed to write block data to ancient store err="the append operation is out-order"
```

Further, I was in the `console` at the time and after the `Crit` close, it seems my tty is borked. May be another issue.

**Comment by meowsbits** [comment]
Possibly rel https://github.com/ethereum/go-ethereum/issues/21245

**Comment by meowsbits** [comment]
Reported by @BelfordZ 

![belfordz-snot](https://cdn.discordapp.com/attachments/644711692655132672/758205162182672414/unknown.png)

**Comment by meowsbits** [comment]
Closing following #245 as fixed by #263
